Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m

```diff
@@ -254,22 +249,25 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
 
 - (void)scrollViewDidScroll:(UIScrollView *)scrollView
 {
-    
     NSArray *visibleRows = [self.tableView visibleCells];
     UITableViewCell *firstVisibleCell = [visibleRows objectAtIndex:0];
     NSIndexPath *path = [self.tableView indexPathForCell:firstVisibleCell];
     
-    
     //Scroll to the selected date.
     NSDate *toDate = [NSDate dateFromString:self.sortedDateKeys[path.section] ];
     
-    NSDateComponents *firstComponents = [[NSCalendar currentCalendar] components:NSCalendarUnitDay | NSCalendarUnitMonth | NSCalendarUnitYear fromDate:toDate];
     
-    NSInteger year = [firstComponents year];
-    NSInteger month = [firstComponents month];
-    NSInteger day = [firstComponents day];
+    if (![toDate isEqualToDate:self.currentDateViewed]){
```

Comment:
Let's evaluate this boolean outside of the conditional and using a variable name that makes this conditional read better. Perhaps something similar to
```
BOOL selectedDateIsCurrentlyViewed
if (!selectedDateIsCurrentlyViewed) ...
```